utils/chain-ops/genesis/testdata/test-deploy-config-devnet-l1.json

```diff
@@ -21,7 +23,6 @@
   "cliqueSignerAddress": "0xca062b0fd91172d89bcd4bb084ac4e21972cc467",
 
   "protocolVaultRecipient": "0xBcd4042DE499D14e55001CcbB24a551F3b954096",
-  "proposerRewardVaultRecipient": "0x71bE63f3384f5fb98995898A86B02Fb2426c5788",
```

Comment:
Why did you remove `proposerRewardVaultRecipient` instead of `validatorRewardVaultRecipient`?